### ISSUE TITLE
Cannot index files compressed with gzip, please use bgzip

### ISSUE THREAD

**Issue by replikation** [nanoporetech/medaka#126]
**Describe the bug**
* it seems that there are some issues related to gzip bgzip in medaka_consensus
* reproduced on 0.11.4 and 0.11.2

**Logging**
* command used:
```bash
medaka_consensus -ireads.fastq -d assembly.fasta -o polished -t 24 -m r941_min_high_g340_rle
```
* it fails with
```bash
  Constructing minimap index.
  [E::fai_build3_core] Cannot index files compressed with gzip, please use bgzip
  [faidx] Could not build fai index draft.fastrle.gz.fai
```
* i use .fastq and .fasta as input (not gziped); therefore it might be related to this code part [here](https://github.com/nanoporetech/medaka/blob/5b22d1eef03aa826856ea671e847c2f95e369476/scripts/medaka_consensus#L96) which uses gzip instead of bgzip ?
* both of these .gz files are created for me during medaka_consensus; then I get the error message above

**Environment (if you do not have a GPU, write No GPU):**
 - Installation method [conda] (complete new environment inside docker)
 - OS: Ubuntu 18.04
No GPU

**Additional context**

**Comment by cjw85** [comment]
Hi @replikation,

Thanks for reporting this, we will get this changed. You are correct that the files should be compressed with `bgzip` in the `medaka_consensus` program rather than `gzip`.

While we fix this, you can simply amend the `medaka_consensus` file to use `bgzip`. The best way to do this depends on how you have installed medaka, the simplest way which will work for all our supported methods is to simply open up the file in the installed location and edit in place:

    which medaka_consensus | xargs -I {} sed -i 's/gzip/bgzip/' {}

**Comment by cjw85** [comment]
This has been fixed in v0.11.5.

**Comment by hkaspersen** [comment]
Hello! I am using version 2.1.0 and I am encountering this error. 
`[E::fai_build3_core] Cannot index files compressed with gzip, please use bgzip`

Seems to be a problem in this version as well?
